exporter/elasticsearchexporter/elasticsearch_bulk.go

```diff
@@ -175,7 +175,10 @@ func createElasticsearchBackoffFunc(config *RetrySettings) func(int) time.Durati
 	}
 }
 
-func shouldRetryEvent(status int) bool {
+func shouldRetryEvent(status int, retryOnStatus []int) bool {
+	if retryOnStatus == nil {
```

Comment:
```suggestion
	if len(retryOnStatus) == 0 {
```
Or no need for the `if` at all.

Reply:
oh, I forgot slice typed nils are safe to range over. Nice catch.